bootstrap.yml

```diff
@@ -70,3 +70,21 @@
         vars:
           lab_vars: "{{ (lab_name == 'scale') | ternary(scale, alias) }}"
         when: lab_name in ['scale', 'alias'] and virtual_uc != true
+
+      # use of a local time server is essential or at least a good idea for Ceph, Kerberos, Redis, 
+      # maybe Kubernetes, and other components sensitive to server time skew
+ 
+      - block:
+        - name: default to corporate time server if lab not recognized
+          set_fact:
+            time_servers: ["clock.redhat.com", "clock2.redhat.com"]
+
+        - name: use bos.redhat.com time servers for Alias lab in Westford
+          set_fact:
+            time_servers: ["clock1.bos.redhat.com", "clock2.bos.redhat.com"]
+          when: lab_name = "alias"
```

Comment:
use == here for conditional check instead of =

Reply:
did as you suggested